src/components/People/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,98 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+
+import { Person } from '../../../types';
+import { findParent } from '../../../helpers';
+import { PeopleLink } from '../PeopleLink';
+import { SortType } from '../../../enums/SortType';
+import { SortLink } from '../../SortLink';
+
+type Props = {
+  people: Person[],
+  personId: string,
+};
+
+export const PeopleTable: FC<Props> = (props) => {
+  const {
+    people,
+    personId,
+  } = props;
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          {[...Object.keys(SortType)]
```

Comment:
Spread is redundant here. Object keys already creates a new array.
```suggestion
          {Object.keys(SortType)
```